examples/ftp-edi-message-to-salesforce-opportunity/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "integration_samples"
```

Comment:
```suggestion
org = "salesforce_samples"
```

Reply:
apply where necessary

Reply:
updated in [5452761](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/319/commits/54527610e82929dda4ebc8f90016a66c39eb1501)